core/src/main/java/hudson/lifecycle/UnixLifecycle.java

```diff
@@ -50,12 +52,12 @@
  * @since 1.304
  */
 public class UnixLifecycle extends Lifecycle {
```

Comment:
My inclination is to deprecate and maybe remove all these implementations except `ExitLifecycle`. The server-side analogue to https://github.com/jenkinsci/jep/pull/373#issuecomment-892853164

Reply:
I concur. Perhaps a topic of investigation for future PRs.